packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -117,6 +117,10 @@ export class StripeWebhookHandler extends StripeHandler {
         case 'plan.deleted':
           await this.handlePlanDeletedEvent(request, event);
           break;
+        case 'tax_rate.created':
+        case 'tax_rate.updated':
+          await this.handleTaxRateCreatedOrUpdatedEvent(request, event);
+          break;
```

Comment:
Based on the docs and my own examination of what you can do for a tax rate in the Stripe dashboard, there is no `tax_rate.deleted` event.

Reply:
There's also a type-check on this, so trying to put that in would fail to compile if it doesn't exist.